oss_fuzz_integration/runner.py

```diff
@@ -305,37 +412,70 @@ def introspector_run(
     else:
         build_project(project_name, to_clean=True)
         setup_next_corpus_dir(project_name)
-    
-    # Build sanitizers with introspector
-    build_project(project_name, sanitizer="introspector") 
 
-    # get the latest corpus
-    latest_corpus_dir = get_recent_corpus_dir()
+    curr_dir = os.path.abspath(".")
 
-    # copy over inpsoector and coverage reports
+    if get_project_lang(project_name) == 'jvm':
```

Comment:
I'd prefer to put the language specific items into the oss-fuzz environment as much as we can. Can the logic that follows be put in the base-builder?